src/plugins/usage_collection/server/index.ts

```diff
@@ -17,17 +17,14 @@
  * under the License.
  */
 
-import { PluginInitializerContext } from 'kibana/server';
-import { UsageCollectionPlugin } from './plugin';
+import { UsageCollectionCollectorOptions, UsageCollectionCollector } from '../../../core/server';
+import { UsageCollectionPlugin, UsageCollectionPluginSetup } from './plugin';
 
 export {
-  AllowedSchemaTypes,
-  MakeSchemaFrom,
-  SchemaField,
-  CollectorOptions,
-  Collector,
-} from './collector';
-export { UsageCollectionSetup } from './plugin';
+  UsageCollectionPluginSetup as UsageCollectionSetup,
```

Comment:
Using `UsageCollectionPluginSetup` because it has the `@deprecated` flag, encouraging to move to the core service.